### PR TITLE
Feature/frank03

### PR DESCRIPTION
**Pull request by dellaert**
Likelihood model struct and compiling EM pipeline. 
Next: test on real data

### CODE REVIEW

File: Sources/SwiftFusion/Geometry/Pose2.swift [hunk 1]
```diff
@@ -63,6 +63,25 @@ public struct Pose2: Codable, LieGroup, Equatable, KeyPathIterable {
   @differentiable public var rot: Rot2 { coordinate.rot }
 }
 
+// FRANK: Perturb randomly
+extension Pose2 {
+  public mutating func perturbWith(stddev v: Vector3,
+                                   seed: TensorFlowSeed = Context.local.randomSeed
```
Comment (ProfFan) [inline Sources/SwiftFusion/Geometry/Pose2.swift:69]: Gotta put in my notes :)

**Review by ProfFan — APPROVED** [review]

**Comment by dellaert** [comment]
@ProfFan , while @marcrasi and @dabrahams might be doing a closer look, could you try and run this on real data to make sure the script works? I am not set up to do so.

**Comment by ProfFan** [comment]
Sure, I'll try :)

**Comment by dellaert** [comment]
I merged, hope @ProfFan will take care of numpy issue in CI in a different PR, or disable the offending tests.

**Pull request merged**